HW02_A_Graph-Fails.Rmd

```diff
@@ -51,36 +54,48 @@ ggplot(data = mpg) +
 ### Graph Fail 3
 Two mistakes in this graph. First, I wanted to make the the points slightly bolder, but changing the alpha to 2 does nothing. What does alpha do and what does setting it to 2 do? What could be done instead if I want the points slightly bigger? 
 
+  - Alpha only changes opacity, ranging from 0 to 1. Setting it to 2 won't do anything. in order to make the points bigger, I changed the size argument    to 2.
+
 Second, I wanted to move the legend on top of the graph since there aren't any points there, putting it at approximately the point/ordered pair (5, 40). How do you actually do this? Also, how do you remove the legend title ("class")? Finally, how would you remove the plot legend completely?
+
+  - in order to move the legend , I changed the vector position of the argument legen.position to c(0.6, 0.85). In order to remove the legend title        "class", I added theme(legend.title = element_blank()). To remove the legend completely, I added theme(legend.position = 'none')

```

Comment:
Something I am unclear about is if setting it equal to 'none' is the same as setting it equal to element_blank()????